app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
You should inherit the `Dwarf` class from `ABC` as well because it does not implement all abstract methods from the `Player` class.